test/gtest/ucs/test_rcache.cc

```diff
@@ -202,31 +206,31 @@ static uintptr_t virt_to_phys(uintptr_t address)
     fd = open(pagemap_file, O_RDONLY);
     if (fd < 0) {
         ucs_error("failed to open %s: %m", pagemap_file);
-        pa = -1;
+        pa = UINTPTR_MAX;
```

Comment:
use std::numeric_limits instead of C macros

Reply:
ok, fixed